src/views/AddItem.jsx

```diff
@@ -15,7 +16,7 @@ export function AddItem() {
 	const [formData, setFormData] = useState(initialState);
 	const { itemName, estimate } = formData;
 
-	// for now, we retrieve the token for the test list from local storage;
+	// retrieves token from local storage, if one exists
 	const token = window.localStorage.getItem('tcl-shopping-list-token');
```

Comment:
@tmflower this is similar to our conversation around the `currentTime` variable this morning. Currently, if a user uses your new, handy dandy button and goes to the Home Screen, this component has already loaded and your `token` variable maintains its original value. I believe if you move the `token` variable to within the body of the component, this variable will be updated each time the user navigates to the page.

Reply:
Thanks for reviewing, @wheninseattle! I'm confused because the token IS inside of the component as of the most recent commit (the one called "minor refactor"). I was experimenting with putting it outside, which didn't work how I wanted it do, but I forgot to put it back before I made the commit and pushed. So then I fixed it, committed and pushed again. Which I thought meant you would all be working with the most updated version of the code. I can see the code with the most recent commit, and it runs correctly for me both locally and with the live link. Do you only see one commit for this PR?

Reply:
I just noticed that this PR was in draft mode, which I'm guessing it reverted to after I made that second commit. Maybe that was the issue. Hopefully you'll see the 2nd commit now.

Reply:
How odd! Well, it is entirely possible that when I reviewed it, I had not yet pulled down the most recent changes. In any case, it looks great now!

Reply:
😅